### ISSUE TITLE
powerdns-4.1.0 in dns01 appliance breaks Danube Cloud

### ISSUE THREAD

**Issue by dn0** [erigones/esdc-factory#113]
```
Fatal error: Trying to set unknown parameter 'allow-recursion'
```

PowerDNS was updated to 4.1 in pkgsrc 2017Q4:
https://github.com/joyent/pkgsrc/commits/joyent/release/2017Q4/net/powerdns/distinfo

The 4.1 changed the way recursor works:
https://doc.powerdns.com/authoritative/changelog/4.1.html#change-4.1.0

This affects new Danube Cloud installations from 3.0.0-beta2 upwards.

**Comment by dn0** [comment]
We will create custom PowerDNS 4.0 packages for 2016Q4 and 2017Q4.

Additionally, the PowerDNS SQL backends switched to prepared statements so we have to change configuration of pgBoucer on mgmt.

**Comment by dn0** [comment]
Closed by commit e017ff3. Will be fixed in 3.0.0-rc3.
